unit-tests/test-phase.js

```diff
@@ -224,4 +223,19 @@ describe('phase', () => {
       assert.isFunction(phase.triggerGC);
     });
   });
+
+  describe('timeoutError', () => {
+    it('error thrown', () => {
+      let errorFunc = () => {
+        phase.timeoutError();
+      };
+      assert.throw(errorFunc, 'Test timeout exceeded ' + OPTIONS.timeout + 'ms');
```

Comment:
Alright so all the other tests that have 

    let errorFunc = () => {

don't have whitespace after, so you're saying you want whitespace after all of those? Thats fine I just want to be consistent